business/iter8.go

```diff
@@ -172,7 +206,18 @@ func (in *Iter8Service) ParseJsonForCreate(body []byte) (string, error) {
 	object.Spec.TrafficControl.MaxTrafficPercentage = newExperimentSpec.TrafficControl.MaxTrafficPercentage
 	object.Spec.TrafficControl.MaxIterations = newExperimentSpec.TrafficControl.MaxIterations
 	object.Spec.TrafficControl.TrafficStepSize = newExperimentSpec.TrafficControl.TrafficStepSize
+	object.Spec.TrafficControl.Interval = newExperimentSpec.TrafficControl.Interval
 	object.Spec.Analysis.AnalyticsService = "http://iter8-analytics.iter8:" + strconv.Itoa(in.GetAnalyticPort())
+	rr, err := in.GetIter8RoutingReferences(newExperimentSpec.Namespace, newExperimentSpec.Service)
```

Comment:
I wouldn't make any call inside a Parse() method.
it can be added as parameter, but calls should be done from the business method, otherwise we are mixing concerns and then it's harder to follow.

Reply:
understand. thank you